src/styles/main.scss

```diff
@@ -1,7 +1,727 @@
+      }
+
+      .description {
+        line-height: 15px;
+        margin-top: 24px;
+        margin-bottom: 48px;
+        &__master {
+          font-size: 1rem;
+          font-weight: 700;
+          color: #d12d35;
+        }
+
+        &__duration {
+          font-size: 1rem;
+          font-weight: 400;
+          color: #161616;
+        }
+      }
+
+      .main__section-row__second {
+        display: grid;
+        grid-template-columns: repeat(3, 0fr);
+        column-gap: 24px;
+        justify-content: center;
+
+        .img5,
+        .img6,
+        .img7 {
+          @include changeSize($mobile) {
+            width: 88px;
+            height: 87px;
+          }
+        }
+      }
+
+      .main__section-row__third {
+        display: grid;
+        grid-template-columns: repeat(2, 0fr);
+        column-gap: 24px;
+        justify-content: center;
+
+        .img8,
+        .img9 {
+          @include changeSize($mobile) {
+            width: 130px;
+            height: 130px;
+          }
+        }
+      }
+    }
+
+    .section__view-button {
+      position: absolute;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      color: #fff;
+      width: 324px;
+      height: 56px;
+      font-family: "Cinzel", sans-serif;
+      background-color: #d12d35;
+      border: 1px solid transparent;
+      margin-top: 40px;
+
+      &:hover {
+        cursor: pointer;
+        background-color: #ae1c23;
+      }
+
+      &:active {
+        background: #eb343d;
+      }
+    }
+  }
+
+  &__section-3 {
+    margin-top: 235px;
+    @include changeSize($mobile) {
+      padding: 20px 20px;
+      width: 280px;
+    }
+
+    .explore {
+      width: 500px;
+      margin: 0 auto;
+
+      &__title {
+        font-size: 48px;
+        font-weight: 700;
+        text-align: center;
+        letter-spacing: -1.5px;
+
+        @include changeSize($mobile) {
+          width: 280px;
+          font-size: 32px;
+          letter-spacing: -1.5px;
+        }
+      }
+    }
+
+    .section__block {
+      margin-top: 48px;
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      column-gap: 24px;
+      justify-content: center;
+
+      @include changeSize($mobile) {
+        grid-template-columns: 1fr;
+      }
+
+      .section__col-4 {
+
+        @include changeSize($mobile) {
+          padding: 20px 20px;
+          width: 280px;
+        }
+
+        .section__photos {
+          width: 324px;
+          height: 320px;
+          transition: transform 0.3s ease-in;
+
+          &:hover {
+            cursor: pointer;
+            transform: scale(1.1);
+          }
+
+          @include changeSize($mobile) {
+            width: 288px;
+            height: 287px;
+          }
+        }
+
+        .title__area {
+          font-size: 24px;
+          color: #d12d35;
+          font-weight: 700;
+          text-align: center;
+          margin-bottom: 10px;
+          box-sizing: border-box;
+        }
```

Comment:
does not belong nested here - must be at 0 indendation level and outside `the { ... } block`